modules/@apostrophecms/modal/ui/apos/components/AposModalConfirm.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <AposModal
     :modal="modal" class="apos-confirm"
-    @esc="cancel" @no-modal="$emit('safe-close')"
+    v-on="mode !== 'alert' ? { 'esc': cancel } : null"
```

Comment:
Alerts have no cancel

Reply:
Or only cancel? Either way, it's the same thing and we want clearing to be intentional.

Reply:
Whichever way we go on my suggestion that this be a true confirmation saving for you on "OK," adding alert support is still sweet.

Reply:
True. That's a good idea.